dev/com.ibm.ws.cdi.extension.bce_fat/fat/src/com/ibm/ws/cdi/extension/bce_fat/BuildCompatibleExtensionsTest.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.extension.bce_fat;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.ws.cdi.extension.bce_fat.basic.app.BasicBCEExtension;
+import com.ibm.ws.cdi.extension.bce_fat.basic.app.BasicBCETestServlet;
+import com.ibm.ws.cdi.extension.bce_fat.ear.lib.EarBCEExtension;
```

Comment:
If at some point someone adds some more tests then using application/package names like "basic" and "ear" might not be descriptive enough? However, the best I can think of at the moment is "earBCETest" which isn't all that much better :-(
```suggestion
import io.openliberty.cdi.internal.extension.bce.fat.earBCETest.lib.EarBCEExtension;
```

Reply:
`basicear` and `basicwar` would probably be ok for the moment. These are just tests to ensure that BCEs work at all within .wars and .ears, they're not testing any specific required functionality.

Reply:
Updated the package names to `io.openliberty.cdi40.internal.fat.bce.{basicwar, basicear}`